spec/features/discover_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+describe 'as a user on the discover page' do
```

Comment:
SUPER minor, but maybe we change this describe to 'as a REGISTERED user...' just so it's clear

Reply:
Added!